src/cpu/arm.rs

```diff
@@ -172,13 +180,46 @@ fn detect_features() -> u32 {
     features
 }
 
+#[cfg(all(target_os = "ios", target_arch = "aarch64"))]
+fn detect_features() -> u32 {
+    let mut features = ARMCAP_STATIC;
+
+    extern "C" {
+        fn sysctlbyname(
+            name: *const u8,
+            oldp: *mut core::ffi::c_void,
+            oldlenp: *mut crate::c::size_t,
+            newp: *mut core::ffi::c_void,
+            newlen: crate::c::size_t,
+        ) -> core::ffi::c_int;
```

Comment:
`core::ffi::c_int` is too new for *ring* to use because of the MSRV, so you have to use `c::int`. (`c_void` is fine to use.)

I think we should just use the libc crate to call this, if that works. We have a goal to avoid libc dependencies but it seems like this function is provided by the OS's libc so we can't get around libc anyway, just like with `getauxval` on Linux/Android. And hopefully using the libc crate to call it will handle any/all linking-related aspects.